[03-Mar-2023 15:21:21 UTC] Uncaught exception: 'Exception' with message: 'No route matched.'
Stack trace: #0 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('favicon.ico')
#1 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Router.php' on line 92
[03-Mar-2023 15:21:37 UTC] Uncaught exception: 'Exception' with message: 'No route matched.'
Stack trace: #0 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('favicon.ico')
#1 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Router.php' on line 92
[03-Mar-2023 15:21:39 UTC] Uncaught exception: 'Exception' with message: 'No route matched.'
Stack trace: #0 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('favicon.ico')
#1 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Router.php' on line 92
[03-Mar-2023 15:39:43 UTC] Uncaught exception: 'PDOException' with message: 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'eee@mail.com' for key 'users.email''
Stack trace: #0 C:\laragon\www\SignUp-MVC\App\Models\User.php(64): PDOStatement->execute()
#1 C:\laragon\www\SignUp-MVC\App\Controllers\Signup.php(20): App\Models\User->save()
#2 [internal function]: App\Controllers\Signup->createAction()
#3 C:\laragon\www\SignUp-MVC\Core\Controller.php(27): call_user_func_array(Array, Array)
#4 C:\laragon\www\SignUp-MVC\Core\Router.php(81): Core\Controller->__call('create', Array)
#5 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('signup/create')
#6 {main}
Throw in 'C:\laragon\www\SignUp-MVC\App\Models\User.php' on line 64
[03-Mar-2023 15:41:40 UTC] Uncaught exception: 'Exception' with message: 'Method successAction not found in controllerApp\Controllers\Signup'
Stack trace: #0 C:\laragon\www\SignUp-MVC\Core\Router.php(81): Core\Controller->__call('success', Array)
#1 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('signup/success')
#2 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Controller.php' on line 31
[03-Mar-2023 16:58:52 UTC] Uncaught exception: 'ErrorException' with message: 'Undefined property: App\Models\User::$name'
Stack trace: #0 C:\laragon\www\SignUp-MVC\App\Models\User.php(19): Core\Error::errorHandler(2, 'Undefined prope...', 'C:\\laragon\\www\\...', 19)
#1 C:\laragon\www\SignUp-MVC\App\Models\User.php(52): App\Models\User->validate()
#2 C:\laragon\www\SignUp-MVC\App\Controllers\Signup.php(20): App\Models\User->save()
#3 [internal function]: App\Controllers\Signup->createAction()
#4 C:\laragon\www\SignUp-MVC\Core\Controller.php(27): call_user_func_array(Array, Array)
#5 C:\laragon\www\SignUp-MVC\Core\Router.php(81): Core\Controller->__call('create', Array)
#6 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('signup/create')
#7 {main}
Throw in 'C:\laragon\www\SignUp-MVC\App\Models\User.php' on line 19
[03-Mar-2023 16:59:37 UTC] Uncaught exception: 'PDOException' with message: 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'eee@gmail.com' for key 'users.email''
Stack trace: #0 C:\laragon\www\SignUp-MVC\App\Models\User.php(64): PDOStatement->execute()
#1 C:\laragon\www\SignUp-MVC\App\Controllers\Signup.php(20): App\Models\User->save()
#2 [internal function]: App\Controllers\Signup->createAction()
#3 C:\laragon\www\SignUp-MVC\Core\Controller.php(27): call_user_func_array(Array, Array)
#4 C:\laragon\www\SignUp-MVC\Core\Router.php(81): Core\Controller->__call('create', Array)
#5 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('signup/create')
#6 {main}
Throw in 'C:\laragon\www\SignUp-MVC\App\Models\User.php' on line 64
[03-Mar-2023 17:02:36 UTC] Uncaught exception: 'PDOException' with message: 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'eee@mail.com' for key 'users.email''
Stack trace: #0 C:\laragon\www\SignUp-MVC\App\Models\User.php(64): PDOStatement->execute()
#1 C:\laragon\www\SignUp-MVC\App\Controllers\Signup.php(20): App\Models\User->save()
#2 [internal function]: App\Controllers\Signup->createAction()
#3 C:\laragon\www\SignUp-MVC\Core\Controller.php(27): call_user_func_array(Array, Array)
#4 C:\laragon\www\SignUp-MVC\Core\Router.php(81): Core\Controller->__call('create', Array)
#5 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('signup/create')
#6 {main}
Throw in 'C:\laragon\www\SignUp-MVC\App\Models\User.php' on line 64
[03-Mar-2023 17:17:38 UTC] Uncaught exception: 'Exception' with message: 'No route matched.'
Stack trace: #0 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('favicon.ico')
#1 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Router.php' on line 92
[03-Mar-2023 17:17:41 UTC] Uncaught exception: 'Exception' with message: 'No route matched.'
Stack trace: #0 C:\laragon\www\SignUp-MVC\public\index.php(32): Core\Router->dispatch('favicon.ico')
#1 {main}
Throw in 'C:\laragon\www\SignUp-MVC\Core\Router.php' on line 92
